app/api/v1/endpoints/message_feedback_endpoints.py

```
@@ -0,0 +1,100 @@
+
+from app.core.settings import settings
+
+try:
+    from app.api.v1.endpoints.auth import get_current_user_id
+except ImportError:
+    # Only allow fallback in development mode - fail loudly in production
+    if settings.is_production_mode():
+        raise ImportError(
+            "Authentication module required in production. "
+            "Cannot import get_current_user_id from app.api.v1.endpoints.auth"
+        )
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)